RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -0,0 +1,126 @@
+    template <typename Vec, typename TAcc = Acc<typename Vec::Dim>>
+    ALPAKA_FN_HOST ALPAKA_FN_INLINE WorkDiv<typename Vec::Dim> createWorkDiv(const Vec& blocksPerGrid,
+                                                                             const Vec& threadsPerBlock,
+                                                                             const Vec& elementsPerThreadArg) {
+      Vec adjustedBlocks = blocksPerGrid;
+      Vec adjustedThreads = threadsPerBlock;
+
+      // special overrides for CPU/host cases
+      if constexpr (std::is_same_v<Platform, alpaka::PlatformCpu>) {
+        adjustedBlocks = Vec::all(static_cast<Idx>(1));
+
+        if constexpr (alpaka::accMatchesTags<TAcc, alpaka::TagCpuSerial>) {
+          // Serial execution, set threads to 1 as well
+          adjustedThreads = Vec::all(static_cast<Idx>(1));  // probably redundant
+        }
+      }
+
+      return WorkDiv<typename Vec::Dim>(adjustedBlocks, adjustedThreads, elementsPerThreadArg);
+    }
+
+    // The constants below are usually used in functions like alpaka::math::min(),
+    // expecting a reference (T const&) in the arguments. Hence,
+    // ALPAKA_STATIC_ACC_MEM_GLOBAL needs to be used in addition to constexpr.
+
+    // 15 MeV constant from the approximate Bethe-Bloch formula
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kMulsInGeV = 0.015;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kMiniMulsPtScaleBarrel[6] = {
+        0.0052, 0.0038, 0.0034, 0.0034, 0.0032, 0.0034};
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kMiniMulsPtScaleEndcap[5] = {0.006, 0.006, 0.006, 0.006, 0.006};
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kMiniRminMeanBarrel[6] = {
+        25.007152356, 37.2186993757, 52.3104270826, 68.6658656666, 85.9770373007, 108.301772384};
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kMiniRminMeanEndcap[5] = {
+        130.992832231, 154.813883559, 185.352604327, 221.635123002, 265.022076742};
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float k2Rinv1GeVf = (2.99792458e-3 * 3.8) / 2;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kR1GeVf = 1. / (2.99792458e-3 * 3.8);
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kSinAlphaMax = 0.95;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float ptCut = PT_CUT;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kDeltaZLum = 15.0;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kPixelPSZpitch = 0.15;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kStripPSZpitch = 2.4;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kStrip2SZpitch = 5.0;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kWidth2S = 0.009;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kWidthPS = 0.01;
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float kPt_betaMax = 7.0;
+    // Since C++ can't represent infinity, lst_INF = 123456789 was used to represent infinity in the data table
+    ALPAKA_STATIC_ACC_MEM_GLOBAL constexpr float lst_INF = 123456789.0;
```

Comment:
@fwyzard Even though this is a relatively straightforward change and we mean to change it in the future, it requires us to update the data in cms-data/RecoTracker-LSTCore (requiring another PR review and merge) and update the bot here to use the right data. This is because this value is used as a boundary value in those files. Given this complexity, would it be okay if, for the time being, we rename this variable to something that does not imply infinity but how it is actually used (`kVerticalModuleSlope` would be our proposal) and leave this for a follow-up PR?